src/serverMain/kotlin/com/studo/campusqr/Server.kt

```diff
@@ -117,6 +120,7 @@ suspend fun main() {
           post("delete") { call.getAuthenticatedCall()?.deleteLocation() }
           post("editSeatFilter") { call.getAuthenticatedCall()?.editSeatFilter() }
           post("deleteSeatFilter") { call.getAuthenticatedCall()?.deleteSeatFilter() }
+          get("pollLiveCheckIns") { call.pollLiveCheckIns() }
```

Comment:
what is the difference between "get" and "poll"? --> path&function name: `getLiveCheckIns`?